.github/workflows/build.yml

```diff
@@ -102,6 +102,46 @@ jobs:
         path: images.tar
         retention-days: 1
 
+  tekton-pipelines:
+    name: Tekton Tests
+    runs-on: ubuntu-latest
+    env:
+      DOCKER_HOSTNAME: "localhost:5000"
+      DOCKER_NAMESPACE: m4d-system
+    steps:
```

Comment:
m4d-system now renamed to fybrik-system

Reply:
Updated this.  I think it still "works" with an arbitrary name, since this just corresponds to the local registry namespace we are pushing a pulling images from... but we should be consistent with the rest of the tests.